module/settings.js

```diff
@@ -1,4 +1,153 @@
+import DG from "./config.js";
+
+const { ApplicationV2, HandlebarsApplicationMixin } = foundry.applications.api;
+
+const SettingForm = class extends HandlebarsApplicationMixin(ApplicationV2) {
+  constructor() {
+    super();
+  }
+
+  static _namespace;
+
+  static get namespace() {
+    return this.constructor._namespace
+  };
+
+  static get settings() {
+    return {}
```

Comment:
Explicitly error here so its clear to devs what they are doing wrong in the future if they extend the class.

```suggestion
    throw new Error("This static getter must be defined by a subclass")
```